tests/Unit/Models/Stub/BaseModelImplementation.php

```diff
@@ -7,9 +7,6 @@
 use Engelsystem\Models\BaseModel;
 use Illuminate\Database\Eloquent\Builder as QueryBuilder;
 
-/**
- * @property string foo
- */
```

Comment:
That whole file can be removed if i see that correctly

Reply:
deleted file. also couldn't find any references